### ISSUE TITLE
CSP unsafe-inline support

### ISSUE THREAD

**Issue by craigfrancis** [chartist-js/chartist#619]
Just debugging some JS in the [contrast-widget](https://github.com/AdaRoseEdwards/contrast-widget) for AdaRoseEdwards, and came across:

https://github.com/gionkunz/chartist-js/blob/develop/dist/chartist.js#L339

```
svg = new Chartist.Svg('svg').attr({
  width: width,
  height: height
}).addClass(className).attr({
  style: 'width: ' + width + '; height: ' + height + ';'
});
```

It looks like the `width` and `height` is being set twice, the second one using an inline style attribute (which is considered as [unsafe-inline](https://www.w3.org/TR/CSP2/#directive-style-src) when you're applying a CSP header).

Am I missing something, or can that just be changed to:

```
svg = new Chartist.Svg('svg').attr({
  width: width,
  height: height
}).addClass(className);
```

**Comment by gionkunz** [comment]
Hi there, yeah somehow I missed the comment on this line sorry. The reason for this was a bug in Firefox which did not behave correctly on width and height only set as attributes on the SVG element in some cases. We'd need to re-validate if that's still an issue and remove if it works without the style properties across all supported browsers.

**Comment by craigfrancis** [comment]
Thanks @gionkunz, realise browser support can be difficult... I would be interested to know if/when it was fixed in Firefox, as 44.0.2 seems to be ok without it (well, in this case at least).

**Comment by gionkunz** [comment]
Closing this issue in favor of #620
